src/tools/rbd/action/Bench.cc

```diff
@@ -26,6 +26,8 @@ namespace {
 enum io_type_t {
   BENCH_READ = 0,
   BENCH_WRITE,
+
+  BENCH_LAST,
```

Comment:
Thanx @trociny and @dillaman , but from my experience in linux kernel development, put the invalid enum in the end is a good idea.
(1), don't make 0 as UNKNOWN, making 0 as a start point of valid number looks more nature.
(2), BENCH_LAST, can not only tell us the last one, example 2 in this case, is a invalid value, but also it can tell us any one larger than BENCH_LAST is invalid.
(3), BENCH_LAST can also tell us how many io_types we are supporting, because the value of it is the count of the io_type, 2 in my case. Even when we expand our rbd bench for mix mode, then BENCH_LAST would be changed to 3, because it's the last one in our enum definition.

But, It seems that you think that's confusing to you, what about renaming it to IO_TYPE_NUM. and still put it at the last.

Then we can get lots of information from this enum.
(1), if (io_type == IO_TYPE_NUM) -> invalid io_type
(2), if (io_type > IO_TYPE_NUM) -> invalid io_type
(3), the number of io_types we are supporting currently is IO_TYPE_NUM.